sc/src/test/java/uk/ac/ebi/atlas/home/JsonLatestExperimentControllerWIT.java

```diff
@@ -0,0 +1,72 @@
+package uk.ac.ebi.atlas.home;
```

Comment:
Please, space lines following our conventions (see other tests). Same thing applies to `LatestExperimentDaoIT.java`.

Reply:
This is hardly enough.

Again, have a look at other tests and use them as a template. You should be more meticulous. Having readable, maintainable, clean code is our top priority here. You’ll be saving two minutes but adding more than that to future Atlas developers. Please, install the CheckStyle plug-in in IDEA, since that will help you with style fixes.